src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map((styleString) => styleString.split(':').map((rule) => rule.trim()))
```

Comment:
The `map` function is correctly splitting each style string into key-value pairs, but it assumes that each string will contain exactly one colon. Consider adding error handling for cases where the string does not contain a colon or contains multiple colons.